.travis.yml

```diff
@@ -1,12 +1,15 @@
 language: ruby
 before_install:
-  - gem update --system
-  - gem install bundler
+#  - gem update --system
```

Comment:
can you remove this commented out line please?

Reply:
maybe squash it and the first two commits into one ... or everything into one commit

Reply:
Done